packaging_uom/__manifest__.py

```diff
@@ -0,0 +1,22 @@
+# Copyright 2015-2019 ACSONE SA/NV (<http://acsone.eu>)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+{
+    "name": "Packaging UOM",
+    "version": "12.0.1.0.0",
+    "author": 'ACSONE SA/NV, '
+              'Odoo Community Association (OCA)',
+    "category": "Warehouse",
+    "development_status": "Production/Stable",
+    "maintainers": ["rousseldenis"],
+    "website": "http://github.com/stock-logistics-warehouse",
```

Comment:
https://github.com/stock-logistics-warehouse would be nice.

Reply:
Done !